docs/HOWTO_DEVELOPMENT_LOCAL_SETUP.md

```diff
@@ -1,13 +1,44 @@
 ## Getting started
 
+---
+### [Repository](https://github.com/MoveOnOrg/Spoke)
+1. If you have not already, Fork this repo then clone your forked copy. Then future pull requests can be made from your repo to Spoke. 
+```
+git clone <url of your forked repo>
+```
+
+2. change your pwd (present working directory) to the spoke directory before installations 
+```
+cd spoke
+```
+
+3. You may use this opportunity to set the remote upstream to spoke's repo for future fetches.
+```
+git remote add upstream https://github.com/MoveOnOrg/Spoke.git
+``` 
+   -  you can check that this is configured correctly to push to the origin and fetch from spoke's repo. 
+   ```
+   git remote -v
+   ```
+   Your origin and Upstream should appear configured correctly.
+   - this is a good time to take a look at [Syncing a Fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) for full understanding.
 
+---
 ### Downloading
 
 1. Install the Node version listed in `.nvmrc`. [NVM](https://github.com/creationix/nvm) is one way to do this (from the spoke directory):
    ```
-   nvm install
-   nvm use
+   nvm install 12
+   nvm use 12
```

Comment:
`nvm install` and `nvm use` will install and use the node version specified in the `.nvmrc` file (https://github.com/MoveOnOrg/Spoke/blob/main/.nvmrc), so there is no need to add the `12` to those commands

Reply:
Good to know, I appreciate your input here. Completed.